packages/pluggableWidgets/badge-native/package.json

```diff
@@ -16,7 +16,9 @@
     "lint": "..\"/../../node_modules/.bin/eslint\" --config ../../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
     "test": "pluggable-widgets-tools test:unit:native",
     "test:e2e:local:ios": "pluggable-widgets-tools test:e2e:native:local:ios",
+    "test:e2e:local:ios:debug": "pluggable-widgets-tools test:e2e:native:local:ios",
```

Comment:
This is the exact same command as the one above it

Reply:
Ah, I see what we've done now. We test against the actual command in one of our scripts, to determine what to do. I have to say, that is a setup waiting for confusion and trouble, but ok...